utils.js

```diff
@@ -0,0 +1,79 @@
+//@ts-check
+const defaultGroups = {
+	'accessor-pairs': [{ 'accessorPair': true, 'sort': 'alphabetical' }],
+	'accessors': [{ 'kind': 'get', 'accessorPair': false, 'sort': 'alphabetical' }, { 'kind': 'set', 'accessorPair': false, 'sort': 'alphabetical' }],
+	'methods': [{ 'type': 'method', 'sort': 'alphabetical' }],
+	'private-accessor-pairs': [{ 'name': '/(_|#).+/', 'accessorPair': true, 'sort': 'alphabetical' }],
+	'private-accessors': [{ 'name': '/(_|#).+/', 'kind': 'get', 'accessorPair': false, 'sort': 'alphabetical' }, { 'name': '/(_|#).+/', 'kind': 'set', 'accessorPair': false, 'sort': 'alphabetical' }],
+	'private-methods': [{ 'name': '/(_|#).+/', 'type': 'method', 'sort': 'alphabetical' }],
+	'private-properties': [{ 'name': '/(_|#).+/', 'type': 'property', 'sort': 'none' }],
+	'properties': [{ 'type': 'property', 'sort': 'none' }],
+	'static-methods': [{ 'type': 'method', 'sort': 'alphabetical', 'static': true }],
+	'static-properties': [{ 'type': 'property', 'sort': 'none', 'static': true }]
+};
+
+const defaultOrder = [
+	'[static-properties]',
+	'[static-methods]',
+	'[properties]',
+	'constructor',
+	'[accessor-pairs]',
+	'[accessors]',
+	'[methods]',
+	'[private-properties]',
+	'[private-accessor-pairs]',
+	'[private-accessors]',
+	'[private-methods]'
+];
+
+export const getSortMemberRules = (order, groups) => {
```

Comment:
Should this utility be documented in the README, or is it just used internally? If only internal, then all the more reason to export the others from `index.js` but not this one.